internal/scorecard/alpha/scorecard.go

```diff
@@ -56,42 +58,56 @@ func (o Scorecard) RunTests() (testOutput v1alpha2.ScorecardOutput, err error) {
 	}
 
 	// create a ConfigMap holding the bundle contents
-	o.bundleConfigMap, err = createConfigMap(o, bundleData)
+	o.BundleConfigMapName, err = createConfigMap(o, bundleData)
```

Comment:
This should not be exported right? Would it make sense for a caller to set this field before calling `RunTests()`?

Reply:
done, I un-exported it, but left the CreateConfigMap() in there to use the same ctx that RunTests is using.